assets/helpers/utilities.js

```diff
@@ -42,3 +42,16 @@ export function clickSubstituteKeyPressHandler(handler?: () => void = () => {})
   };
 }
 
+// Attempts to parse a boolean from a string.
+export function parseBoolean(boolString: ?string): ?boolean {
+
+  switch (boolString) {
+    case 'true':
+      return true;
+    case 'false':
```

Comment:
This seems weird. If we have to explicitly parse strings to bools, can we just `default: false` if not `case 'true'`? Are there cases where you specifically need a `null` back from here?

Reply:
Hmm, I think that was based more on semantics of that function rather than a technical reason why we might need a `null`. If we're explicitly looking for a boolean, and someone passes a garbage string to that function, is it correct to say "oh, this counts as false"? Do we want to go down the path of "truthiness" (a la JavaScript)?

Reply:
You could take a second parameter `default: boolean` which is used if the string isn't true or false

Reply:
Ok, I've changed this.